webapp/locales/de.json

```diff
@@ -22,7 +22,8 @@
     "responsible": "Verantwortlicher gemäß § 55 Abs. 2 RStV ",
     "bank": "Bankverbindung",
     "germany": "Deutschland",
-    "code-of-conduct": "Verhaltenscodex"
+    "code-of-conduct": "Verhaltenscodex",
+    "termsAndConditionsConfirmed": "Ich habe die <a href=\"/terms-and-conditions\" target=\"_blank\">Nutzungsbedingungen</a> durchgelesen und stimme ihnen zu."
   },
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-08-14T09:24:52Z" title="Wednesday, August 14th 2019, 11:24:52 am +02:00">Aug 14, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -22,7 +22,9 @@
     "responsible": "Verantwortlicher gemäß § 55 Abs. 2 RStV ",
     "bank": "Bankverbindung",
     "germany": "Deutschland",
-    "code-of-conduct": "Verhaltenscodex"
+    "code-of-conduct": "Verhaltenscodex",
+    "termsAndConditionsCeckbox": "<a href=\"/terms-and-conditions\" target=\"_blank\">Nutzungsbedingungen</a> gelesen und verstanden",
+    "termsAndConditionsNoChecked": "Bitte lies und bestätige die Nutzungsbednigungen!"
```


two typos:
`termsAndConditionsCeckbox` --> `termsAndConditionsCheckbox`
`termsAndConditionsNoChecked` --> `termsAndConditionsNotChecked`

or, maybe preferable because it gives translators a better idea of the expected sentence:
`termsAndConditionsRead` and `confirmTermsAndConditions`

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-08-14T10:26:56Z" title="Wednesday, August 14th 2019, 12:26:56 pm +02:00">Aug 14, 2019</time>_

---


thx, I changed it after your suggestion.

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-08-14T10:26:56Z" title="Wednesday, August 14th 2019, 12:26:56 pm +02:00">Aug 14, 2019</time>_

---


thx, I changed it after your suggestion.